pipelines/steps/approve-odw-private-endpoints.yaml

```diff
@@ -0,0 +1,17 @@
+parameters:
+  env: ''
+  pythonVersion: ''
+
+# Note: Although this can approve all private endpoints, this is only really needed for Managed private endpoints created by the Synapse deployment
+steps:
+  - script: |
+      set -x
+      python${{ parameters.pythonVersion }} --version
+      export PYTHONPATH="${PYTHONPATH}:/./"
+      python${{ parameters.pythonVersion }} $(System.DefaultWorkingDirectory)/pipelines/scripts/approve_private_endpoints.py --env "${{ parameters.env }}"
```

Comment:
Does this vary ? Maybe using a variable or a central script root might be better ? 
Something like this : 
scriptRoot: $(Build.SourcesDirectory)/pipelines/scripts

Reply:
We can't define a variable at this level. The only way we could do it is by adding a parameter, but this could then result in misuse of the script (by passing the wrong path)